### DISCUSSION TITLE
How to get the drop-down menu for gh pages?

### DISCUSSION THREAD

**Discussion by wslisam** [community/community#22086] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/0e54e858750f7a8ed6078322b03b9635dc0e854d.png" data-download-href="/uploads/short-url/22MB5MOZnbXavmhv4lZ3eS4vHl3.png?dl=1" title="螢幕截圖 2020-07-19 下午5.23.38" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/55217028/181068829-e5b9e73a-b42b-4a8b-93f7-3db3a4c092f3.png" alt="螢幕截圖 2020-07-19 下午5.23.38" data-base62-sha1="22MB5MOZnbXavmhv4lZ3eS4vHl3" width="689" height="274" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/0e54e858750f7a8ed6078322b03b9635dc0e854d_2_689x274.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/0e54e858750f7a8ed6078322b03b9635dc0e854d_2_1033x411.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/0e54e858750f7a8ed6078322b03b9635dc0e854d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/0e54e858750f7a8ed6078322b03b9635dc0e854d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">螢幕截圖 2020-07-19 下午5.23.38</span><span class="informations">1196×476 41.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I follow the online tutorial to deploy me website using react. but my website only show me the Readme.md.</p>
<p>I found that I need to switch the gh pages in setting .<br>
(<a href="https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site" rel="nofollow noopener">https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site</a>)</p>

**Comment by otter-computer** [comment, marked as answer]
<p>GitHub Pages has 2 different types of sites: User/Organization sites and Project sites:</p>
<p><a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites" class="onebox" target="_blank" rel="nofollow noopener">https://docs.github.com/en/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites</a></p>
<p>User/Organization sites are created when you name the repository <code>username.github.io</code> and can only be served from the master branch. From the looks of your screenshot it seems like you’ve created this type of site,  meaning the dropdown menu isn’t available.</p>
<p>If you’d prefer to serve your site from the <code>gh-pages</code> branch then you’ll need to change the type of Pages site to a Project site by renaming your repository to something other than <code>username.github.io</code>.</p>
